### DISCUSSION TITLE
Please make this.

### DISCUSSION THREAD

**Discussion by darkyvr** [community/community#139999] [Other Feature Feedback, Questions, & Ideas]
One of my longest and biggest irritations with github is how difficult it is to actually find the download location of any given project, now for any given programmer it's easy, they'll say oh it's just here n there etc. But for those who know NOTHING about github they'll be going like, oh is the program maybe in  the <> code section? or do i have to scroll down? what are all these push and commit items? Is my download in there?, do i scroll down even more?  

So i suggest the following, just a huge DOWNLOADS button, so that people who don't know anything about github just know-> GO THERE->DOWNLOADS to get the latest download executable for the program, rather than constantly scrolling through the page, downloading the wrong items etc etc, i really hope this gets resolved because i'm sure so many people who aren't programmers, who don't want or need to go down the entire github rabbit hole, and just want to simply download the latest release of a any given program can just get that easily.

![downloads](https://github.com/user-attachments/assets/0e06f22d-b715-4f41-8b5f-40660a879c7a)

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @darkyvr ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
